frontend/src/api/visitors/averageSessionsService.js

```diff
@@ -60,3 +60,5 @@ export const averageSessionService = {
     fetchChartValues,
     fetchTableValues
 };
+
+export default averageSessionService;
```

Comment:
if you going to export this kind of default, you no need to export const above

Reply:
removed all export default ...Service;